inputs.css

```diff
@@ -0,0 +1,29 @@
+.form-input-size {
```

Comment:
This is not a generic `input` style but a specific one for the filtering form. It shouls tay in `style.css`